src/ORM/Table.php

```diff
@@ -2304,4 +2305,28 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * This can be used to validate any other array as well.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateCount($value, $expectedCount = 0, $operator = '>')
```

Comment:
I'm OK with changing that. But what really annoys me is that this took now SIX DAYS to come to this conclusion after I've explicitly asked for exactly that on IRC and in the previous issue and PR.

What can we do to speed up this process of discussing things without spending *days* or even weeks on getting something done?